source/_drafts/Update-on-the-state-of-Tox.md

```diff
@@ -0,0 +1,87 @@
+happened and Tox continues to progress, so there's a lot to talk about.
+
+#Tox core
+
+[toxcore](https://github.com/irungentoo/toxcore) has been largely in maintenance mode,
+with memory leaks, logic errors and other bugs being fixed every day. Recently
+work has begun on implementing a [new, cleaner API](https://github.com/irungentoo/toxcore/tree/new_api)
+written by the [Tox4j](https://github.com/sonOfRa/tox4j) developers.
+
+Additionally, work is progressing on the complete [groupchat re-design](https://github.com/JFreegman/toxcore)
+and overhaul, started by [alnf](https://github.com/alnf) with currently being
+lead primarily by [JFreegman](https://github.com/JFreegman). Features include moderation
+abilities and group chat persistence (!). These group chats are currently a long
+way from being ready to merge into master; however, there is currently a properly
+[modified build of Toxic](https://jenkins.libtoxcore.so/job/toxic_linux_beta/lastSuccessfulBuild/artifact/toxic)
+for GNU-Linux available with the changes. Note that important packet IDs have been changed
+from the primary Tox network for testing purposes, so don't use that binary as your regular
+client. Please also continually update it if you use it, because breaking changes
+to the new groupchats are continually being made as they undergo primary
+development, and old nodes on the testing network can make testing difficult for
+updated nodes with newer code. You can follow development on the #tox-groupchats
+IRC channel on Freenode.
+
+#Audio filtering and echo cancellation
+
+It's been around for a few months now, but [uTox](https://github.com/notsecure/uTox)
+has had audio filtering, provided by [code](https://github.com/irungentoo/filter_audio) from [WebRTC](https://code.google.com/p/webrtc/).
+([qTox](https://github.com/tux3/qTox) theoretically has had the same code, but due
+to a bug, did not. That was bug was fixed yesterday.)
+
+Recently though, [irungentoo](https://github.com/irungentoo) (project founder and
+lead developer, in case you needed reminding :) ) has created [some patches for
+OpenAL](https://github.com/irungentoo/openal-soft-tox)
+that allow for cancelling speaker echoes from your microphone's sound input, a
+necessary feature for using a microphone without headphones or earbuds. uTox
+quickly put that to use, and qTox gained the same ability yesterday. Note that,
+if you compile these clients yourself, you will need the patched version of OpenAL
+to link against to enjoy echo cancellation. The official builds of both qTox and uTox
+are both built that way, and if you need you can try using the [OpenAL builds](https://jenkins.libtoxcore.so/search/?q=openal)
+on [Jenkins](https://jenkins.libtoxcore.so/) used for qTox and uTox.
+
+Currently echo cancellation is considered experimental, and Tox is seeking testing
+and feedback from users (on IRC, reddit, or wherever) about how effective it is.
```

Comment:
"wherever" sounds like we don't care about user's feedback. You could just remove it and add another sentence "You can leave your feedback on our [IRC](link) or [Reddit](link)."